modules/rosa-sharing-vpc-dns-and-roles.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-shared-vpc-config.adoc
+:_content-type: PROCEDURE
+[id="rosa-sharing-vpc-dns-and-roles_{context}"]
+= Creating your DNS and cluster creation roles
+
+After the *VPC-owning AWS account* user creates a virtual private cloud, subnets, and an IAM role for sharing the VPC resources, the *cluster-creating AWS account* user must reserve an _openshiftapps.com_ DNS domain and create Operator roles to communicate back to the *VPC-owning AWS account*.
+
+.Prerequisites
+
+* You have the ARN for the IAM role that is used to share your VPC.
+
+.Procedure
+
+. The cluster creator reserves an _openshiftapps.com_ a DNS domain with the following command:
++
+[source,terminal]
+----
+$ rosa create dns-domain
+----
++
+The command creates a reserved _openshiftapps.com_ DNS domain.
++
+[source,terminal]
+----
+I: DNS domain '14eo.p1.openshiftapps.com' has been created.
+I: To view all DNS domains, run 'rosa list dns-domains'
+----
+. After creating the DNS domain, the cluster creator needs to create an OIDC configuration. Review this article for more information link:https://access.redhat.com/articles/7031018[on the ODIC configuration process]. The following command produces the OIDC config ID that you need:
++
+[source,terminal]
+----
+$ rosa create oidc-config
+----
++
+You receive confirmation that the command created an OIDC configuration.
++
+[source,terminal]
+----
+I: To create Operator Roles for this OIDC Configuration, run the following command and remember to replace <user-defined> with a prefix of your choice:
+	rosa create operator-roles --prefix <user-defined> --oidc-config-id 25tu67hq45rto1am3slpf5lq6jargg
+----
+. After the OIDC configuration is created, create the Operator roles using the following command:
++
+[source,terminal]
+----
+$ rosa create operator-roles --oidc-config-id <oidc-config-ID> <1>
+    --installer-role-arn <Installer_Role> <2>
+    --shared-vpc-role-arn <Created_VPC_Role_Arn> <3>
+    --prefix <operator-prefix> <4>
+----
++
+--
+<1> Provide the ODIC configuration ID that you created in the previous step.
+<2> Provide the cluster creator's installer ARN that was created as part of the `rosa create account-roles` process.
+<3> Provide the ARN for the role that the *VPC-owning AWS account* created.
+<4> Provide a prefix for the Operator roles.
+--
++
+[NOTE]
+====
+The Installer account role and the shared VPC role should have a one-to-one relationship. If you wish to create multiple shared VPC roles, you should create one set of account roles per shared VPC role.
+====
+
+When complete, share the _full_ domain name, which is created with `<intended_cluster_name>.<created_dns_domain>`, your _Ingress Operator Cloud Credentials_ and _Installer_ roles' ARN with the *VPC-owning AWS account* user.
```

Comment:
```suggestion
After you create the Operator roles, share the full domain name, which is created with `<intended_cluster_name>.<created_dns_domain>`, your Ingress Operator cloud credentials, and your installer roles' ARN with the *VPC-owning AWS account* user.
```
The highlighting in this sentence makes it hard to understand. Would this revision work?